esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -116,48 +124,44 @@ CThorNodeGroup* CThorNodeGroupCache::lookup(const char* groupName, unsigned time
 
 void CWsDfuEx::init(IPropertyTree *cfg, const char *process, const char *service)
 {
-    StringBuffer xpath;
 
     DBGLOG("Initializing %s service [process = %s]", service, process);
 
     espProcess.set(process);
 
-    xpath.appendf("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/DefaultScope", process, service);
-    cfg->getProp(xpath.str(), defaultScope_);
+    VStringBuffer xpath("Software/EspProcess[@name=\"%s\"]", process);
+    IPropertyTree *processTree = cfg->queryPropTree(xpath);
+    if (!processTree)
+        throw makeStringException(-1, "Invalid WsDFU configuration");
```

Comment:
I'm going to change it to "config not found for process '%s'"
purely because a number of the other services error out like that.
But looks like a lot of the services are inconsistent in this area. Some don't check (and would crash), others check and skip silently, or others error out with different text.

It would better if all validated in 1 common place - but not for this PR.